src/components/settings/ReportsTab.vue

```diff
@@ -60,11 +112,32 @@ export default {
   data() {
     return {
       config: this.configItem,
+      reportLogo: false,
+      logoPath: "",
+      chosenFile: null,
     };
   },
+  created() {
+    if (!this.config.logo) {
+      this.config.logo = {
+        enabled: false,
+        path: {},
```

Comment:
@dockabernathy01 Shouldn’t `path` be an empty string?

Reply:
@dacoaster , I stored the logo path like this : 
 , I have stored the path like this : 
    "path": {
      "path": "E:\\Mine\\Electron\\Yattie-Session\\logo.png",
      "name": "logo.png",
      "size": 1610
    }
So, I think it will be empty object.

Reply:
`name` and `size` don't belong under `path`.

It should be:
```
logo: {
    enabled: BOOLEAN,
    path: STRING,
    name: STRING,
    size: INT
}
````

Reply:
@dacoaster , Ok, I will update the code like that.